gon/angular.py

```diff
@@ -1,14 +1,14 @@
 from robust.angular import (Orientation,
                             orientation as _orientation)
 
-from .base import Point
+from .point import Point
```

Comment:
why?

Reply:
Usually, imports like `from pprint import pprint` and similar are a sign of a bad design or a bad naming. I'm not actually opposed to keeping the class `Point` in the module `point`, but then you'd probably want to expose them in the `__init__`. So, instead of writing `from gon.polygon import Polygon` we would have `from gon import Polygon`. What do you think?